src/pages/_document.tsx

```diff
@@ -1,57 +1,46 @@
-import { ServerStyleSheet } from "styled-components";
 import {
   Components,
   fetchDecoratorReact,
   Props,
 } from "@navikt/nav-dekoratoren-moduler/ssr";
 import Document, {
   DocumentContext,
+  DocumentInitialProps,
   Head,
   Html,
   Main,
   NextScript,
 } from "next/document";
 import serverEnv from "server/utils/serverEnv";
-import Script from "next/script";
+
+// The 'head'-field of the document initialProps contains data from <head> (meta-tags etc)
+const getDocumentParameter = (
+  initialProps: DocumentInitialProps,
+  name: string
+): string => {
+  return initialProps.head?.find((element) => element?.props?.name === name)
+    ?.props?.content;
+};
 
 export default class MyDocument extends Document<{ Decorator: Components }> {
   static async getInitialProps(ctx: DocumentContext) {
-    const sheet = new ServerStyleSheet();
-    const originalRenderPage = ctx.renderPage;
+    const initialProps = await Document.getInitialProps(ctx);
+
     const isAudienceSykmeldt = ctx.pathname.startsWith("/sykmeldt");
 
     const decoratorParams: Props = {
       env: serverEnv.DECORATOR_ENV,
       context: isAudienceSykmeldt ? "privatperson" : "arbeidsgiver",
       chatbot: true,
       redirectToApp: true,
-
       level: "Level4",
     };
 
     const Decorator = await fetchDecoratorReact(decoratorParams);
 
-    try {
-      ctx.renderPage = () =>
-        originalRenderPage({
-          enhanceApp: (App) => (props) =>
-            sheet.collectStyles(<App {...props} />),
-        });
+    const language = getDocumentParameter(initialProps, "lang");
 
-      const initialProps = await Document.getInitialProps(ctx);
-      return {
-        ...initialProps,
-        Decorator,
-        styles: (
```

Comment:
Vil css-in-js funke hvis man fjerner dette?

Reply:
Det vil det. Lagde litt likere oppsett som dinesykmeldte. Ble litt smoothere loading også, slik at menyen lastes instant, og vi får spinner inne i selve "content" området for appen vår